packages/gunshi/src/types.ts

```
@@ -78,7 +78,7 @@ export type CommandArgKeys<A extends Args> = GenerateNamespacedKey<
 /**
  * Command environment.
  */
-export interface CommandEnvironment<A extends Args = Args> {
+export interface CommandEnvironment<A extends Args = Args, E extends Record<string, unknown> = {}> {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)